types/schema/data/Proposal.ts

```diff
@@ -109,14 +106,7 @@ export interface BaseProposal {
   newDwellings?: {
     newBuild?: {count: number};
   };
-  units?: {
-    residential: {
-      development: DevelopmentType;
-      type: UKResidentialUnitType;
-      bedrooms: number;
-      identicalUnits: number;
-    }[];
-  };
```

Comment:
I incorrectly had `development` rather than `tenure` here - which means we can actually share the same "UK Residential Unit" definition between `property` & `proposal` - hence moving this definition to `shared.ts` now.